libraries/Microsoft.Bot.Connector/Authentication/RetryParams.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.Connector.Authentication
+{
+    public class RetryParams
```

Comment:
This class has no unit tests. Seems like it would be very easy to test.

Reply:
All these tests were in V3 and I just forgot to port them :) I didn't put love in to this PR this week with all the ICMs. I'll update it now with all the code and feedback.